.circleci/config.yml

```diff
@@ -183,9 +183,24 @@ jobs:
       - attach_workspace:
           at: ~/repo
       - run:
-          name: Deploy
+          name: Override version
           command: |
-            echo "Not publishing package!"
+            sed -i -e "s/^\(version = \".*\)\"$/\1.dev${CIRCLE_BUILD_NUM}\"/" setup.py
+      - run:
+          name: Authenticate
+          command: |
+            echo "[distutils]" > ~/.pypirc
+            echo "index-servers =" >> ~/.pypirc
+            echo "    jfrog" >> ~/.pypirc
+            echo >> ~/.pypirc
+            echo "[jfrog]" >> ~/.pypirc
+            echo "repository:https://globality.jfrog.io/globality/api/pypi/pypi" >> ~/.pypirc
+            echo "username:$JFROG_USERNAME" >> ~/.pypirc
+            echo "password:$JFROG_PASSWORD" >> ~/.pypirc
+            echo >> ~/.pypirc
+      - run:
+          name: Publish
+          command: python setup.py sdist upload -r jfrog
```

Comment:
This comes from merging master into this branch (see https://github.com/globality-corp/microcosm-flask/pull/319)